models/integration_test/integration_test.yml

```diff
@@ -19,14 +19,6 @@ models:
           tests:
             - not_null
 
-  - name: test_table
```

Comment:
i know @dot2dotseurat uses this sometimes, is it better to leave this and fix the gap to match however ```test_view``` is setup?